clients/web/src/templates/body/systemConfiguration.jade

```diff
@@ -67,7 +67,9 @@ form.g-settings-form(role="form")
               placeholder="Default: #{defaults['core.upload_minimum_chunk_size'] || ''}",
               title="For large files, the minimum size of all but the last chunk.")
           | CORS (Cross-origin resource sharing) allows access from other
-          | websites.
+          | websites. These settings are described in detail
+          a(href="http://girder.readthedocs.org/en/latest/security.html#cors-cross-origin-resource-sharing"
```

Comment:
Can we make this a relative link?  That way if we have multiple versions of the docs on readthedocs (or elsewhere), the versions will stay within themselves.